packages/core/src/routes/admin-user/basics.ts

```diff
@@ -148,6 +169,7 @@ export default function adminUserBasicsRoutes<T extends AuthedRouter>(...args: R
           username,
           name,
           ...conditional(password && (await encryptUserPassword(password))),
+          ...conditional(passwordEncrypted && { passwordEncrypted, passwordEncryptionMethod }),
```

Comment:
Seems to create a user with a specified `password` and `passwordEncryptionMethod`, the encryption method will not take any effect. Should we forbid such cases?

Reply:
This is already implemented, please take a look at line 137